packages/common/src/authorization/organizationMemberAbility.test.ts

```diff
@@ -1638,97 +1699,4 @@ describe('Organization member permissions', () => {
             ).toEqual(false);
         });
     });
-
-    describe('test project preview permissions', () => {
```

Comment:
are these tests no longer relevant ? should we write new ones ?

Reply:
updated